storage/rocksdb/rdb_datadic.cc

```diff
@@ -1690,9 +1691,9 @@ int Rdb_key_def::unpack_record(TABLE *const table, uchar *const buf,
           (const uchar *)unp_reader.read(RDB_CHECKSUM_SIZE));
 
       const uint32_t computed_key_chksum =
-          crc32(0, (const uchar *)packed_key->data(), packed_key->size());
+          my_checksum(0, (const uchar *)packed_key->data(), packed_key->size());
       const uint32_t computed_val_chksum =
-          crc32(0, (const uchar *)unpack_info->data(),
+          my_checksum(0, (const uchar *)unpack_info->data(),
                 unpack_info->size() - RDB_CHECKSUM_CHUNK_SIZE);
```

Comment:
The `(const uchar *)` casts should be unnecessary and should be removed.

Reply:
done. sorry missed it.